int32 angle0
int32 angle1
int32 angle2
int32 angle3
int32 angle4
int32 angle5
---
bool angle_accepted